src/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen, fireEvent } from '@testing-library/react';
+import Calculator from '../components/Calculator';
```

Comment:
- [ ] The test is failing due to a typo in the path. It should be `../components/calculator`. The `c` should be small.